packages/overlay/src/popper.ts

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// Bundle only what we want from popper
+// See: https://popper.js.org/docs#popper-lite-tree-shaking
+
+import {
+    popperGenerator,
+    defaultModifiers,
+} from '@popperjs/core/lib/popper-lite';
+import { Instance } from '@popperjs/core/lib/types';
+import { Placement } from '@popperjs/core/lib/enums';
+
+import flip from '@popperjs/core/lib/modifiers/flip';
+import preventOverflow from '@popperjs/core/lib/modifiers/preventOverflow';
+import arrow from '@popperjs/core/lib/modifiers/arrow';
+import offset from '@popperjs/core/lib/modifiers/offset';
+import { computeArrowRotateStyles } from './popper-arrow-rotate';
+
+export const createPopper = popperGenerator({
+    defaultModifiers: [
+        ...defaultModifiers,
+        flip,
+        preventOverflow,
+        arrow,
+        offset,
+        computeArrowRotateStyles,
+    ],
+});
+
+interface WindowNodeWorkaround {
+    process: {
+        env?: {
+            NODE_ENV?: string;
+        };
+    };
+}
+const windowNodeWorkaround = window as WindowNodeWorkaround;
+
+// Popper code references process.env
+if (!windowNodeWorkaround.process) {
+    windowNodeWorkaround.process = {
+        env: {
+            NODE_ENV: 'production',
+        },
+    };
+}
```

Comment:
Where did the popper team land on this? Do we want to do this workaround, or just specify usage of the babel plugin for consumers and in our es-dev-server? I'm honestly not sure what the best approach is here.

Reply:
Oh, yah. I let this ball drop. They still seem to be building the "esm" version as `.mjs` files and they don't have Typescript typings. So, we should continue to use the "lib" versions. I guess that means your babel plug-in solution is probably the way to go. I will update this.